packages/esm-patient-list-management-app/src/lists-table/use-pagination-info.ts

```diff
@@ -0,0 +1,32 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+export function usePaginationInfo(pageSize: number, totalItems: number, pageNumber: number, currentItems: number) {
+  const { t } = useTranslation();
+
+  const pageSizes = useMemo(() => {
+    let numberOfPages = Math.ceil(totalItems / pageSize);
+    if (isNaN(numberOfPages)) {
+      numberOfPages = 0;
+    }
+
+    return [...Array(numberOfPages).keys()].map((x) => {
+      return (x + 1) * pageSize;
+    });
+  }, [pageSize, totalItems]);
+
+  const numberOfItemsDisplayed = useMemo(() => {
+    if (pageSize > totalItems) {
+      return `${totalItems} / ${totalItems}`;
+    } else if (pageSize * pageNumber > totalItems) {
+      return `${pageSize * (pageNumber - 1) + currentItems} / ${totalItems}`;
+    } else {
+      return `${pageSize * pageNumber} / ${totalItems}`;
+    }
+  }, [pageSize, totalItems, pageNumber, currentItems]);
+
+  return {
+    pageSizes,
+    itemsDisplayed: `${numberOfItemsDisplayed} ${t('items', ' items')}`,
```

Comment:
It would be better to do this as a templated string, I think, to accomodate RTL languages.

Reply:
Sorry, @ibacher, how should that look like?

Reply:
This:

```suggestion
    itemsDisplayed: t('itemsDisplayed', '{{numberOfItemsDisplayed}} items', { numberOfItemsDisplayed }),
```

Reply:
Does it need to be escaped? This is what gets rendered:

![CleanShot 2024-01-16 at 12  06 56@2x](https://github.com/openmrs/openmrs-esm-patient-management/assets/8509731/166820a9-dbc9-460c-a717-fc3037f150e8)

Reply:
`{{- numberOfItemsDisplayed}}` [based on this](https://www.i18next.com/translation-function/interpolation#unescape).

Reply:
This syntax (where we're explicit about unescaping the interpolated value by setting the `escapeValue` property to false) ended up working for me:

`i18next.t('keyEscaped', { myVar: '<img />', interpolation: { escapeValue: false } })`

Using the `{{- key }}` syntax did not.